.github/workflows/main.yaml

```diff
@@ -25,7 +24,7 @@ jobs:
 
       # Pulls all versions of this container. This allows the docker build to use the cache for faster builds.
```

Comment:
Is this comment still accurate? If not, update to reflect new behavior. Same in pr.yaml.